src/ml_board/frontend/Dashboard/src/worker_socket/MainThreadCallbacks.ts

```diff
@@ -0,0 +1,65 @@
+import { DataFromSocket, DataToRedux } from "./DataTypes";
+import { MLGYM_EVENT } from "./EventsTypes";
+import handleEvaluationResultData, { evalResultCustomData, EvaluationResultPayload } from "./event_handlers/evaluationResultDataHandler";
+import handleExperimentStatusData from "./event_handlers/ExperimentStatusHandler";
+import handleJobStatusData from "./event_handlers/JobStatusHandler";
+
+// Note: I don't like having anything here other than the callbacks but it is what it is :')
```

Comment:
I think it is even good practice to separate things  like this. :-)

Reply:
I have a good news from a future PR, all the variables except the map are removed (not needed) ;)